homeassistant/components/sms/notify.py

```diff
@@ -0,0 +1,43 @@
+"""Support for SMS notification services."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.notify import PLATFORM_SCHEMA, BaseNotificationService
+from homeassistant.const import CONF_NAME
+import homeassistant.helpers.config_validation as cv
+
+from .const import CONF_PHONE_NUMBER, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {vol.Required(CONF_PHONE_NUMBER): cv.string, vol.Optional(CONF_NAME): cv.string}
```

Comment:
All our exists SMS integration use `recipient`. Please adapt this and rename `phone_number`